content/docs/atom-guide/look-dev/materials/material-file-spec.md

```diff
@@ -4,12 +4,55 @@ description: "Material files (`*.material`) are written in JSON format and conta
 toc: true
 ---
 
+## Elements
+
 Material files (`*.material`) are written in JSON format and contain the following elements.
 
+### **description** (*optional*)  
+Provides a description or comment from the material's author.
+
+### **materialType**
+The path to the material type. Materials must reference a material type which provides the list of available properties and the shaders to use for rendering. The path must be relative to the asset root or to the material file.
+
+### **materialTypeVersion** (*optional*)  
+Indicates the version number of the material type that was used to create this material. This can be used by the material type version update feature to support backward compatibility with older versions of the material type. (See [Material Type Specification](material-type-file-spec)'s [`version`](material-type-file-spec/#version) and [`versionUpdates`](material-type-file-spec/#versionupdates)).
+
 ### **parentMaterial** (*optional*)  
-The path to the parent material file. If specified, the material inherits the properties of another parent material. The parent material must have the same material type as the material. The path must be relative to the asset root or to the material file. If not specified, the material inherits directly from the materialType. 
+The path to the parent material file. If specified, the material inherits the properties of another parent material. The parent material must have the same material type as the material. The path must be relative to the asset root or to the material file. If not specified, the material inherits default values directly from the materialType. 
+
+### **propertyValues**
+Lists the names and values to set for material properties. Any properties that aren't specified here will inherit the value from the parent material if available or from the material type's default value. The JSON data type used for the values should match the property's data type as specified in the .materialtype file. However, some data type conversion is supported (i.e. "1" can be used to set a float property to "1.0", etc.).
```

Comment:
Per Style Guide, replacing abbreviations and Latin phrases for full English words. I don't think it's needed, but you can add "and so on", or similar, to the end of the example if you'd like. 
```suggestion
Lists the names and values to set for material properties. Any properties that aren't specified here will inherit the value from the parent material if available or from the material type's default value. The JSON data type used for the values should match the property's data type as specified in the `.materialtype` file. However, some data type conversion is supported (for example, "1" can be used to set a float property to "1.0").
```